boa_engine/src/syntax/ast/node/object/mod.rs

```diff
@@ -344,6 +347,24 @@ pub enum PropertyName {
     Computed(Node),
 }
 
+impl PropertyName {
+    pub(in crate::syntax) fn literal(&self) -> Option<Sym> {
+        if let Self::Literal(sym) = self {
+            Some(*sym)
+        } else {
+            None
+        }
+    }
+
+    pub(in crate::syntax) fn prop_name(&self) -> Option<Sym> {
```

Comment:
```suggestion
    pub(in crate::syntax) fn name(&self) -> Option<Sym> {
```
I think its better to call it just `name()`

Reply:
This function should be the start of this spec function https://tc39.es/ecma262/#sec-static-semantics-propname.
That's where the names comes from.
I will put a reference to the spec in the docs for that function to make that clear.